modules/imgimport/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
-include_directories(include ../core/include)
+include_directories(include ../core/include ../core/src)
```

Comment:
To fix the undefined reference errors, you should link the Core library to the ImageImport library using target_link_libraries
see [targetid/CMakeLists.txt](https://github.com/UWARG/computer-vision/blob/master/modules/targetid/CMakeLists.txt) for an example.